### PR TITLE
[SITES-570] add transition links to root

### PR DESCRIPTION
**Pull request by ltankey**
image

### CODE REVIEW

File: app/views/templates/root_node.html.haml [hunk 1]
```diff
@@ -32,6 +32,28 @@
   %a.see-more More about Australia
 
   %section.home-news-list
```
Comment (tessereth) [inline app/views/templates/root_node.html.haml]: Should it be styled with the home-news-list class or something else?

Comment (ltankey) [inline app/views/templates/root_node.html.haml]: Nope

**Comment by tessereth** [comment]
LGTM

**Comment by bjgordon** [comment]
LGTM, but amend the commit message.

**Pull request merged**